CHANGES.adoc

```diff
@@ -3,6 +3,9 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+== 0.1.151 Dec 2 2020
+- update model to 0.0.101
```

Comment:
There are a bunch more changes to add here :)
```
~/g/s/g/o/ocm-sdk-go (⎇ master) $ git log v0.1.150..HEAD --oneline --no-merges --no-decorate --reverse 
0b51bf9 Move logging code to `logging` package
9733ca7 Rename `Metrics` to `MetricsSubsystem`
efcfb1b Add method to read metrics subsystem
0903d9f Load metrics subsystem configuration from file
c881fc8 Load string if it doesn't look like a file
eec027b Load `os.Stdin` in dump configuration example
08d16f0 Reject URLs without scheme or host name
eaa2d83 Add redirection tests
```

Reply:
damn, my bad, ill add them now